apps/condo/domains/organization/schema/ResetOrganizationService.test.js

```diff
@@ -179,4 +202,41 @@ describe('ResetOrganizationService', () => {
 
     })
 
+    test('notification not send to resident after reset organization', async () => {
+        const client = await makeClientWithServiceConsumer()
+        const { property, organization, serviceConsumer, resident } = client
+        const [resource] = await MeterResource.getAll(admin, {})
+
+        const [meter] = await createTestMeter(admin, organization, property, resource, {
+            accountNumber: serviceConsumer.accountNumber,
+            unitName: resident.unitName,
+            verificationDate: dayjs().add(-1, 'year').toISOString(),
+            nextVerificationDate: undefined,
+        })
+
+        await createTestMeterReportingPeriod(admin, organization, {
+            notifyStartDay: 1,
+            notifyEndDay: Number(dayjs().format('DD')),
+        })
+
+        await waitFor(async () => {
+            setAllFeatureFlags(true)
```

Comment:
It's better to add this at the beginning of all tests, and there is also "setFeatureFlag" to enable the desired feature